src/components/SingleDatePicker/DatePicker.js

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable no-unexpected-multiline */
```

Comment:
Had to disable this eslint rule because it was giving errors locally when running a build.